CHANGELOG.md

```diff
@@ -0,0 +1,262 @@
+# v2.0.0
```

Comment:
Il faudrait ajouter un titre au fichier (`# Changelog of AgentJ`) et changer le niveau de titre des numéros de versions (`##` au lieu de `#`).